model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/weights_inferable_quantizers/weights_lut_symmetric_inferable_quantizer.py

```diff
@@ -0,0 +1,150 @@
+        BaseKerasInferableQuantizer
+    from model_compression_toolkit.quantizers_infrastructure.keras.quantizer_utils import lut_quantizer
+
+
+    @mark_quantizer(quantization_target=QuantizationTarget.Weights,
+                    quantization_method=[QuantizationMethod.LUT_SYM_QUANTIZER],
+                    quantizer_type=None)
+    class WeightsLUTSymmetricInferableQuantizer(BaseKerasInferableQuantizer):
+        """
+        Class for quantizing weights using a lut symmetric quantizer
+        """
+
+        def __init__(self,
+                     num_bits: int,
+                     cluster_centers: np.ndarray,
+                     threshold: List[float],
+                     per_channel: bool,
+                     channel_axis: int = None,
+                     input_rank: int = None):
+            """
+            Initialize the quantizer with the specified parameters.
+
+            Args:
+                num_bits: number of bits to use for quantization
+                cluster_centers: the cluster centers to assign the weights
+                threshold: threshold for quantizing weights
+                per_channel: whether to use per-channel quantization
+                channel_axis: axis along which to apply per-channel quantization
+                input_rank: number of dimensions of input tensor the quantizer quantizes
+            """
+
+            super(WeightsLUTSymmetricInferableQuantizer, self).__init__()
+
+            assert isinstance(threshold, list), f'Expected threshold to be of type list but is {type(threshold)}'
+            assert all([isinstance(x, (float, np.float32, np.float64)) for x in
+                        threshold]), f'Expected threshold list to contain float or np.float values but found ' \
+                                     f'{[type(x) for x in threshold]}'
+
+            self.threshold = np.asarray(threshold)
+
+            if per_channel:
+                assert input_rank is not None, f'Input rank is missing in per channel quantization'
+                assert channel_axis is not None, f'Channel axis is missing in per channel quantization'
+                assert len(threshold) >= 1, f'In per-channel quantization threshold list should be of length >= 1 ' \
+                                            f'but is {len(threshold)} '
+            else:
+                assert len(threshold) == 1, f'In per-tensor quantization threshold should be of length 1 but is' \
+                                            f' {len(threshold)}'
+                self.threshold = self.threshold[0]
+
+            self.num_bits = num_bits
+            self.cluster_centers = cluster_centers
+
+            self.per_channel = per_channel
+            self.channel_axis = channel_axis
+            self.input_rank = input_rank
+
+            # Tensorflow's fake_quant_with_min_max_vars_per_channel only works on last axis, so
+            # need to move the quantization axis to the last axis
```

Comment:
Do we actually need to move the channels to the last axis? we're not using TF fake_quant function. Can remove input_rank and all the permutataion if that's true